.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,15 @@
+# yamllint disable-line rule:document-start
 blank_issues_enabled: false
 contact_links:
   - name: 📄Documentation addition/change
     about: 🔨What needs documenting? ✨
+    # yamllint disable-line rule:line-length
```

Comment:
I would rather configure yaml lint to have a longer line limit than keep disabling it. Is that possible?

Reply:
Yes, it is possible to make it a warning instead of disabling it.